<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Simulation</class>
 <widget class="QWidget" name="Simulation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>510</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>580</width>
    <height>510</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>580</width>
    <height>510</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Simulation</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../icons/ncc.png</normaloff>../icons/ncc.png</iconset>
  </property>
  <widget class="QPushButton" name="pushAdd">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>480</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushRemove">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>480</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </widget>
  <widget class="QListView" name="listAvailable">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>221</width>
     <height>441</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushApply">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>480</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QListView" name="listApplied">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>30</y>
     <width>221</width>
     <height>441</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushLoad">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>480</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
